Add the 'Getting started with DateBox' article (#1004)
DevExpress/devextreme-documentation: concepts/05 Widgets/DateBox/00 Getting Started with DateBox/05 Create the DateBox.md

    import { DxDateBoxModule } from 'devextreme-angular';

    @NgModule({
        declarations: [
            AppComponent
        ],
        imports: [
            BrowserModule,
            DxDateBoxModule
        ],
        providers: [ ],
        bootstrap: [AppComponent]
    })
    export class AppModule { }

##### Vue

    <!-- tab: App.vue -->
    <template>
        <DxDateBox
            <!-- Configuration goes here -->
        />
    </template>

    <script>
    import 'devextreme/dist/css/dx.common.css';
    import 'devextreme/dist/css/dx.light.css';

    import { DxDateBox } from 'devextreme-vue/date-box';

    export default {
        components: {
            DxDateBox
        }
    }
    </script>

##### React

    <!-- tab: App.js -->
    import React from 'react';

    import 'devextreme/dist/css/dx.common.css';
    import 'devextreme/dist/css/dx.light.css';

    import { DateBox } from 'devextreme-react/date-box';

    class App extends React.Component {
        render() {
            return (
                <DateBox      
                    // Configuration goes here
                />
            );
        }
    }
    export default App;

---

> If you run this code, you should see the **DateBox** that allows users to set date only. We will let users set time in the next step.

Reviewer: ```suggestion
If you run this code, you will see that the **DateBox** allows users to set the date only. Users will be allowed to set the time in the next step.
```
